src/elements/content-sidebar/__tests__/__snapshots__/SidebarFileProperties.test.js.snap

```diff
@@ -118,6 +118,7 @@ exports[`elements/content-sidebar/SidebarFileProperties render() should render r
       >
         <dl
```

Comment:
we should be able to target this with getByRole 
https://stackoverflow.com/a/76826101